pkgs/shells/nix-bash-completions/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "0.1";
+  name = "nix-bash-completions-${version}";
+
+  src = fetchFromGitHub {
+    owner = "hedning";
+    repo = "nix-bash-completions";
+    rev = "v${version}";
+    sha256 = "1gb6fmnask1xmjv5j5x0jb505lyp0p4lx2kbibfnb2gi57wapxaz";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/bash-completion/completions
+    cp _* $out/share/bash-completion/completions
+  '';
+
+  meta = {
```

Comment:
Just a note: if attrset had `with stdenv.lib;`, you could have omitted it from attributes:

```
meta = with stdenv.lib; {
  license = licenses.bsd3;
  platforms = platforms.all;
  maintainers = with maintainers; [ hedning ];
}
```

If you already know that, nevermind.

Reply:
Thanks, that make sense. Made both changes in a new PR.